OnStreetParking/examples/example.json.csv

```diff
@@ -1,2 +1,2 @@
-"id_", "type_", "category__0_", "category__1_", "category__2_", "allowedVehicleType_", "chargeType__0_", "requiredPermit__0_", "requiredPermit__1_", "permitActiveHours__blueZonePermit_", "maximumAllowedStay_", "availableSpotNumber_", "totalSpotNumber_", "extraSpotNumber_", "dateModified_", "location__type_", "location__coordinates__0__0__0_", "location__coordinates__0__0__1_", "location__coordinates__0__1__0_", "location__coordinates__0__1__1_", "location__coordinates__0__2__0_", "location__coordinates__0__2__1_", "location__coordinates__0__3__0_", "location__coordinates__0__3__1_", "location__coordinates__0__4__0_", "location__coordinates__0__4__1_", "areaServed_", "refParkingGroup__0_", "refParkingGroup__1_"
-"santander:daoiz_velarde_1_5", "OnStreetParking", "blueZone", "shortTerm", "forDisabled", "car", "temporaryFee", "blueZonePermit", "disabledPermit", "Mo, Tu, We, Th, Fr, Sa 09:00-20:00", "PT2H", "3", "6", "2", "2016-06-02T09:25:55.00Z", "Polygon", "-3.80356167695194", "43.46296641666926", "-3.803161973253841", "43.46301091092682", "-3.803147082548618", "43.462879859445884", "-3.803536474744068", "43.462838666196674", "-3.80356167695194", "43.46296641666926", "Zona Centro", "daoiz-velarde-1-5-main", "daoiz-velarde-1-5-disabled"
\ No newline at end of file
+"id_", "type_", "category__0_", "category__1_", "category__2_", "allowedVehicleType_", "chargeType__0_", "requiredPermit__0_", "requiredPermit__1_", "permitActiveHours__blueZonePermit_", "maximumAllowedStay_", "availableSpotNumber_", "occupiedSpotNumber_", "totalSpotNumber_", "extraSpotNumber_", "dateModified_", "location__type_", "location__coordinates__0__0__0_", "location__coordinates__0__0__1_", "location__coordinates__0__1__0_", "location__coordinates__0__1__1_", "location__coordinates__0__2__0_", "location__coordinates__0__2__1_", "location__coordinates__0__3__0_", "location__coordinates__0__3__1_", "location__coordinates__0__4__0_", "location__coordinates__0__4__1_", "areaServed_", "refParkingGroup__0_", "refParkingGroup__1_"
```

Comment:
this file is generated automatically so it is not necessary (either convenient to be in the PR)